docs/content/latest/reference/configuration/yb-master.md

```diff
@@ -341,6 +335,24 @@ The number of replicas, or copies of data, to store for each tablet in the unive
 
 Default: `3`
 
+##### --yb_num_shards_per_tserver
+
+The number of shards per YB-TServer for each YCQL table when a user table is created.
+
+Default: `-1` (server internally sets default value). For servers with two or less CPU cores, then the default value is `4`. For four or more CPU cores, the default value is `8`. If set using `yb-ctl` or `yb-docker-ctl`, then the default value is `2`.
+
+##### --ysql_num_shards_per_tserver
+
+The number of shards per YB-TServer for each YSQL table when a user table is created.
```

Comment:
Since you've mentioned yb-ctl and yb-docker-ctl in the CQL gflag: This flag is set to 2 as well for local cluster installations.

Reply:
Got it.